src/main/webapp/scripts/sys/isoFipsCountryMapScript.js

```diff
@@ -0,0 +1,82 @@
+/**
+ * CU Generic ISO/FIPS Country modification
+ */
+const ISO_COUNTRY_CODE_FIELD = 'document.newMaintainableObject.isoCountryCode';
+const ISO_COUNTRY_NAME_FIELD = 'document.newMaintainableObject.isoCountry.name.div';
+const FIPS_COUNTRY_CODE_FIELD = 'document.newMaintainableObject.fipsCountryCode';
+const FIPS_COUNTRY_NAME_FIELD = 'document.newMaintainableObject.fipsCountry.name.div';
+const EMPTY_STRING = '';
+var previousFipsCountryCode = "";
+var previousIsoCountryCode = "";
```

Comment:
Are these previous-value variables needed? I don't see any areas in this file that are actively reading their values; they're only being written to.